boot/boot_images.cmake

```diff
@@ -59,6 +61,53 @@ ${_efisys_file} 1
 set(ISO_MANUFACTURER "ReactOS Foundation")  # For both the publisher and the preparer
 set(ISO_VOLNAME      "ReactOS")             # For both the Volume ID and the Volume set ID
 
+# ISO image options
+set(ISO_COMMON_OPTIONS
+    -iso-level 4 -publisher ${ISO_MANUFACTURER} -preparer ${ISO_MANUFACTURER} -volid ${ISO_VOLNAME} -volset ${ISO_VOLNAME})
+
+set(ISO_LAYOUT_OPTIONS
+    -duplicates-once -no-cache-inodes -graft-points)
+
+## "El Torito" ISO boot options.
+#
+set(ISO_BOOT_OPTIONS)
+set(ISO_BOOT_REGTEST_OPTIONS) # Useful only for BootCDRegTest
+
+# BIOS-based PC boot entry (x86/x64 only)
+if(ARCH STREQUAL "i386" OR ARCH STREQUAL "amd64")
+    set(ISO_BOOT_OPTIONS
+        -eltorito-platform x86 -eltorito-boot loader/isoboot.bin -no-emul-boot -boot-load-size 4)
```

Comment:
I think `x86` is the default option, do we need to explicitly specify it?
Also please take care, the order of these things can matter according to the documentation I found online:
> If  the  option -eltorito-platform appears before	the first -el-torito-boot option, it sets the platform id for the default boot entry.
 If the option -eltorito-platform appears after an -eltorito-boot option and sets the platform id to a value  different from  the previous value, it starts a new set of boot entries.

https://www.freebsd.org/cgi/man.cgi?query=mkisofs&sektion=8&format=html

Reply:
Yep that's expected. And explicitly specify the default should only clarify things.